test-clients/src/main/resource/bootstrap.properties

```diff
@@ -52,15 +52,13 @@ contracts.allowAutoAssociations=true
 contracts.allowCreate2=true
 contracts.defaultLifetime=7890000
 contracts.localCall.estRetBytes=32
-contracts.maxGas=8000000
+contracts.maxGasPerSec=15000000
```

Comment:
Will `staking.isEnabled` be true by default, so we don't need to update this file ?